.github/pull_request_template.md

```diff
@@ -17,6 +17,7 @@
 ## Types of changes
 <!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
 - [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] Refactor (non-breaking change which refactors the existing code)
```

Comment:
Thanks for adding this. Refactoring is more specific than a non-breaking change.

Please could you change the wording of this to:

`Refactor (non-breaking change which refactors the existing code, improving only its design or structure, and making no changes to its external behaviour)`

Reply:
Actually, never mind. Cancel that. I can make the edit easily, and then I'd like to try out the template.